frontend/common/dispatcher/app-actions.js

```diff
@@ -283,10 +283,11 @@ const AppActions = Object.assign({}, require('./base/_app-actions'), {
       projectId,
     })
   },
-  getOrganisation(organisationId) {
+  getOrganisation(organisationId, force = false) {
```

Comment:
`force` feels a little unclear to me here, can we be more explicit with the variable name? What are we forcing?

Reply:
this change was reverted.